rabies/analysis_pkg/analysis_functions.py

```
@@ -185,48 +185,44 @@
         exists=True, desc=".nii file with spatial components from the fitted prior sources.")
-    NPR_extra_filename = File(
+    CPCA_extra_filename = File(
         exists=True, desc=".nii file with spatial components from the scan-specific extra sources.")
-    optimize_report = traits.Any(
-        exists=False, desc="The NPR optimization report.")
+    CPCA_report = traits.Any(
+        exists=False, desc="The CPCA optimization report.")
 
 
-class NeuralPriorRecovery(BaseInterface):
+class ComplementaryPCA(BaseInterface):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)